lib/elixir/src/elixir_expand.erl

```diff
@@ -27,6 +27,11 @@ expand({'%', Meta, [Left, Right]}, E) ->
 expand({'<<>>', Meta, Args}, E) ->
   elixir_bitstring:expand(Meta, Args, E, false);
 
+expand({'module_sigil', Meta, [Module, String, Modifiers]}, E) ->
```

Comment:
We should call the special form `__sigil__` and it would have to be documented as such.